### ISSUE TITLE
Parsing tables with keys not in quotes

### ISSUE THREAD

**Issue by larsimmisch** [Kilcekru/lua-table#1]
Hi,

I was trying to parse a Lua table like the one below, but it fails with `Unexpected token 'n' at position 3 (line 2:2) near '{\n\tname...`.

This is valid lua as long as the table key is not a lua keyword.
```lua
{
	name = "foo",
	root = "../../..",
	includes = {
		"some/directory",
		"some/other_directory,
	}
}
```

**Comment by larsimmisch** [comment]
See also #2.

**Comment by Kilcekru** [comment]
Thanks for bringing this to my attention.
My use-case does not include this, so i overlooked that this is valid lua.

Also thanks for the pull request. I implemented the fix a little bit different, but included your test cases.

Fix is published with version 1.1.1.
